int32                       id
int32                       sequence
geometry_msgs/Pose2D        center
float64[4]                  covariance
geometry_msgs/Pose          initial_guess